2025-04-18 13:23:54 [    INFO] Starting test session. Logging to: C:\Users\denni\Documents\thoughtful task manager\logs\tests\test_run_2025-04-18_13-23-54.log (conftest.py:70)
2025-04-18 13:24:08 [    INFO] HTTP Request: POST http://localhost:11434/api/chat "HTTP/1.1 200 OK" (_client.py:1038)
2025-04-18 13:24:08 [    INFO] Test passed: test_model (conftest.py:82)
2025-04-18 13:24:08 [    INFO] Test passed: test_client_initialization (conftest.py:82)
2025-04-18 13:24:12 [    INFO] Test passed: test_connection_retry (conftest.py:82)
2025-04-18 13:24:12 [    INFO] Test passed: test_model_list_command (conftest.py:82)
2025-04-18 13:24:12 [    INFO] Test passed: test_model_name_parsing (conftest.py:82)
2025-04-18 13:24:12 [    INFO] Test passed: test_model_verification_failure (conftest.py:82)
2025-04-18 13:24:14 [    INFO] Test passed: test_ollama_service_running (conftest.py:82)
2025-04-18 13:24:14 [    INFO] Test passed: test_task_creation (conftest.py:82)
2025-04-18 13:24:14 [    INFO] Test passed: test_task_to_dict (conftest.py:82)
2025-04-18 13:24:14 [    INFO] Test passed: test_task_from_dict (conftest.py:82)
2025-04-18 13:24:14 [    INFO] Test session completed (conftest.py:72)
